.github/workflows/run-tests.yml

```diff
@@ -204,14 +204,13 @@ jobs:
         run: npm ci
       - name: 'Build'
         run: npm run build
-      - name: 'Netlify Login'
+      - name: 'Prepare Netlify CLI'
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
         run: |
-          # TODO(serhalp) Unpin version once
-          # https://netlify.slack.com/archives/C07686YAY13/p1718139257978049 is resolved.
-          npm i -g netlify-cli@17.26.0
-          netlify login
+          # Control netlify-cli as a regular dev dep but expose it globally for test fixtures to use
```

Comment:
NIT: you could pull this out into a reusable action in github actions.


Something like `.github/actions/netlify`

```

name: 'Install Netlify CLI'
description: some description
runs:
  using: 'composite'
  steps:
    -  run: |
          # Control netlify-cli as a regular dev dep but expose it globally for test fixtures to use
          npm install -g "netlify-cli@$(npm list --json --depth=0 netlify-cli | jq -r ".dependencies[\"netlify-cli\"].version")"
          npx netlify login
```